resources/public/plugin-dev.html

```diff
@@ -305,7 +311,7 @@
         <script src="https://asmblah.github.io/uniter/dist/uniter.js"></script>
         <script src="http://www.skulpt.org/static/skulpt.min.js" type="text/javascript"></script> 
         <script src="http://www.skulpt.org/static/skulpt-stdlib.js" type="text/javascript"></script> 
-    <script src="http://www.biwascheme.org/release/biwascheme-0.6.6.min.js"></script>
+    <script src="http://www.biwascheme.org/release/biwascheme-0.6.6-min.js"></script>
```

Comment:
i think this was a mistake - the .min.js version 404s, the -min.js version exists (and is used by several other files in this repo)